src/lando/main/scm/git.py

```diff
@@ -286,18 +286,25 @@ def process_merge_conflict(
         """Process merge conflict information captured in a PatchConflict, and return a
         parsed structure."""
 
-        failed_re = re.compile(r"patch failed: (.*):\d+", re.MULTILINE)
+        # XXX: The `(?:while searching for)` assertion would be better as `(?!patch
+        # failed)`, but this sort of negative assertion with lookahead prevents the
+        # non-matching group to be captured.
+        failed_re = re.compile(
```

Comment:
Out of scope, but we could compile this outside of the function since it's static to avoid compiling on every call.